README.md

```diff
@@ -1 +1,23 @@
-# portfolio-mobile-version
\ No newline at end of file
+# portfolio-mobile-version
+
+> Project setup for Portfolio mobile version
+
+# Built With
+- HTML
+- CSS
+
+# Authors
+
+:bust_in_silhouette: Author1
+
+- Github: [Github](https://github.com/olawale-o)
+- Twitter: [Twitter](https://twitter.com/ibreaktherules)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/olawale-omoogun-330a051b1/)
+
+# Contributing
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page.](https://github.com/olawale-o/Hello-world/issues)
```

Comment:
- [ ] This issues page redirects to the issues page of another project other than this project. Kindly see to that.